packaging/ubuntu-trusty/debian/dirs

```diff
@@ -0,0 +1,3 @@
+etc/ecs
+var/log/ecs
+var/lib/ecs
```

Comment:
`data` is currently the only thing we use in `var/lib/ecs` so that's probably reasonable to change.  I had just specified the whole of `var/lib/ecs` in case.